fr-FR.lproj/Localizable.strings

```diff
@@ -0,0 +1,369 @@
+/* 
+  Localizable.strings
+  Undecimus
+
+  Created by Brieuc Caillot on 8/26/19.
+  Copyright © 2019 Pwn20wnd. All rights reserved.
+*/
+
+"Bundled Resources version missing." = "La version des ressources groupées est manquante."
+"Unsupported" = "N'est pas prise en charge"
+"Restore RootFS" = "Restaurer RootFS"
+"Ready to restore RootFS" = "Prêt à restaurer RootFS"
+"Re-Jailbreak" = "Re-Jailbreak"
+"Ready to re-jailbreak" = "Ready to re-jailbreak"
+"Jailbreak" = "Jailbreak"
+"Ready to jailbreak" = "Prêt à jailbreak"
+"Error" = "Erreur"
+"Bundled Resources version is missing. This build is invalid." = "La version des ressources groupées est manquante. Le build est invalide."
+"Days" = "Jours"
+"Invalid Entry" = "Entrée non valide"
+"The boot nonce entered could not be parsed" = "Le méthode de signature au démarrage entrée n'a pas pu être analysé"
+"OK" = "OK"
+"The device will be restarted." = "L'appareil va redémarrer."
+"Copy boot nonce?" = "Copier la méthode de signature au démarrage ?"
+"Would you like to copy nonce generator to clipboard?" = "Voulez-vous copier la méthode de signature au démarrage dans le presse papier ?"
+"Yes" = "Oui"
+"No" = "Non"
+"Copy ECID?" = "Copier ECID ?"
+"Would you like to copy ECID to clipboard?" = "Voulez-vous copier ECID dans le presse papier ?"
+"Failed to check for update." = "Impossible de vérifier la mise à jour."
+"An update is available." = "Une mise à jour est disponible."
+"Already up to date." = "Déja à jour."
```

Comment:
Vous êtes à jour.

Reply:
"Failed to check for update." = "mise à jour Impossible a vérifier ."
"An update is available." = "mise à jour est disponible."
"Already up to date." = "aucune mise a jour ."